CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+- (proto)[#1311](https://github.com/evmos/evmos/pull/1311) edit `protocgen.sh` script and generate common types
```

Comment:
nit: I would probably describe this from a user perspective, who should be using the Makefile commands
```suggestion
- (proto)[#1311](https://github.com/evmos/evmos/pull/1311) Also generate common types with `make proto-gen`
```